src/coreComponents/finiteVolume/TwoPointFluxApproximation.cpp

```diff
@@ -247,74 +252,10 @@ void TwoPointFluxApproximation::addToFractureStencil( MeshLevel & mesh,
 
   ArrayOfArraysView< localIndex const > const & fractureConnectorsToFaceElements = fractureSubRegion.m_fractureConnectorEdgesToFaceElements.toViewConst();
 
-  FixedToManyElementRelation const & faceElementsToCells = fractureSubRegion.getToCellRelation();
-
-  localIndex constexpr maxElems = SurfaceElementStencil::maxStencilSize;
-
-  arrayView1d< integer const > const & edgeGhostRank = edgeManager.ghostRank();
-
-  // TODO Note that all of this initialization should be performed elsewhere. This is just here because it was
-  // convenient, but it is not appropriate to have physics based initialization in the flux approximator.
-#if !defined(SET_CREATION_DISPLACEMENT)
-  static_assert( true, "must have SET_CREATION_DISPLACEMENT defined" );
-#endif
-#if SET_CREATION_DISPLACEMENT==1
-  ArrayOfArraysView< localIndex const > const & faceToNodesMap = faceManager.nodeList();
-  arrayView2d< real64, nodes::INCR_DISPLACEMENT_USD > const incrementalDisplacement = nodeManager.incrementalDisplacement();
-  arrayView2d< real64, nodes::TOTAL_DISPLACEMENT_USD > const totalDisplacement = nodeManager.totalDisplacement();
-  arrayView1d< real64 > const aperture = fractureSubRegion.getReference< array1d< real64 > >( "elementAperture" );
-#endif
-
-
-#ifdef GEOSX_USE_SEPARATION_COEFFICIENT
-  arrayView1d< real64 > const apertureF = fractureSubRegion.getReference< array1d< real64 > >( "apertureAtFailure" );
-#endif
-
-#if !defined(ALLOW_CREATION_MASS)
-  static_assert( true, "must have ALLOW_CREATION_MASS defined" );
-#endif
-#if ALLOW_CREATION_MASS==0
-  arrayView1d< real64 > const dens = fractureSubRegion.getReference< array1d< real64 > >( "density_n" );
-#endif
-
-
-#if SET_CREATION_PRESSURE==1
-  arrayView1d< real64 > const fluidPressure_n = fractureSubRegion.getExtrinsicData< extrinsicMeshData::flow::pressure_n >();
-  arrayView1d< real64 > const fluidPressure = fractureSubRegion.getExtrinsicData< extrinsicMeshData::flow::pressure >();
-  // Set the new face elements to some unphysical numbers to make sure they get set by the following routines.
-  SortedArrayView< localIndex const > const newFaceElements = fractureSubRegion.m_newFaceElements.toViewConst();
-  forAll< serialPolicy >( fractureSubRegion.m_newFaceElements.size(), [=]( localIndex const k )
-  {
-    localIndex const kfe = newFaceElements[k];
-#if !defined(SET_CREATION_PRESSURE)
-    static_assert( true, "must have SET_CREATION_PRESSURE defined" );
-#endif
-#if SET_CREATION_PRESSURE==1
-    if( initFlag )
-    {
-      fluidPressure[kfe] = 1.0e99;
-    }
-#endif
-#ifdef GEOSX_USE_SEPARATION_COEFFICIENT
-    apertureF[kfe] = aperture[kfe];
-#endif
-#if !defined(SET_CREATION_DISPLACEMENT)
-    static_assert( true, "must have SET_CREATION_DISPLACEMENT defined" );
-#endif
-#if SET_CREATION_DISPLACEMENT==1
-    if( initFlag )
-    {
-      aperture[kfe] = 1.0e99;
-    }
-#endif
-  } );
-
-#endif
   SortedArray< localIndex > allNewElems;
```

Comment:
is this useful?

Reply:
It's some code for initialization.
It should not be there anymore.
I've removed it!